public/js/cinode-recruitment-public.js

```diff
@@ -34,29 +36,34 @@ jQuery(document).ready(function ($) {
         formData.append("linkedInUrl", linkedInUrl.val());
         formData.append("state", 0);
         formData.append("currencyId", currencyId);
-        if (pipelineId){
+
+        if (pipelineId) {
```

Comment:
`pipelineId` never defined anywhere, is that correct?

Reply:
pipelineId is defined inside form, and its fetched from backend